c/src/detectors/c-insecure-temporary-file-or-directory/c-insecure-temporary-file-or-directory_compliant.c

```diff
@@ -0,0 +1,21 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+// {fact rule=c-insecure-temporary-file-or-directory@v1.0 defects=0}
+#include <stdio.h>
+
+void compliant() {
+
+    // Compliant: Use secure function which manage the generation of unique and secure temporary files automatically.
```

Comment:
Can we shorten multiline comments to a single line? If not, can we maintain same indentation for better readability? Applicable to all files

Reply:
Hi,
Can you check this once. This is a single line comment.